headers/types/common/common.h

```diff
@@ -302,6 +302,40 @@ struct move_data_table {
 };
 ASSERT_SIZE(struct move_data_table, 14534);
 
+// A common structure for pairs of dungeon/floor values
+struct dungeon_floor_pair {
+    uint8_t dungeon_id;
+    uint8_t floor_id;
+};
+ASSERT_SIZE(struct dungeon_floor_pair, 2);
+
+// The adventure log structure.
+struct adventure_log {
+    uint32_t completion_flags[4];
+    uint32_t nb_dungeons_cleared;
+    uint32_t nb_friend_rescues;
+    uint32_t nb_evolutions;
+    uint32_t nb_eggs_hatched;
+    uint32_t successful_steals; //Unused in Sky
+    uint32_t nb_faints;
+    uint32_t nb_victories_on_one_floor;
+    uint32_t pokemon_joined_counter;
+    uint32_t pokemon_battled_counter;
+    uint32_t moves_learned_counter;
+    uint32_t nb_big_treasure_wins;
+    uint32_t nb_recycled;
+    uint32_t nb_gifts_sent;
+    uint32_t nb_big_treasure_wins;
+    uint32_t pokemon_joined_flags[37];
+    uint32_t pokemon_battled_flags[37];
+    uint32_t moves_learned_flags[17];
+    uint32_t items_acquired_flags[44];
+    uint32_t special_challenge_flags;
+    uint32_t footprints_game_points[5];
```

Comment:
Does "footprints" refer to the Sentry Duty minigame? Can we refer to this as `sentry_duty_game_points` so we're sticking with the official terminology? Similarly for the `GetFootprintsGamePoints` and `SetFootprintsGamePoints` functions.